.gitignore

```diff
@@ -1,5 +1,6 @@
 bin
```

Comment:
Please see azure/azure-dev repo for how we should structure having a CLI and extension in the same repo.  Let's buy once cry once for a more organized repo.  I don't want to see this and other random things dropped in the root.  Thanks.